### ISSUE TITLE
Model Training early stopping

### ISSUE THREAD

**Issue by cl3m3nt** [Microsoft/AutonomousDrivingCookbook#43]
Hi, can you provide a feedback to following issue ?
I appreciate your help

## Problem description
There is an early stopping of GPU processing activity during

## Problem details
The issue raised when launching following training activity:
history = model.fit_generator(train_generator, steps_per_epoch=num_train_examples//batch_size, epochs=500, callbacks=callbacks,\
                   validation_data=eval_generator, validation_steps=num_eval_examples//batch_size, verbose=2)
The output message is the following:
Epoch 32/500
Epoch 00032: val_loss did not improve
 - 49s - loss: 0.0022 - val_loss: 3.7090e-04
Epoch 00032: early stopping

This matches only 6% training activity:
Training 6% 32/500 [33:26<6:26:38, 49.57s/it]

## Exepriment/Environment details
* Tutorial used: For example, AirSimE2EDeepLearning
* Environment used: DSVM with K80 GPU on NC6 Azure VM
* Versions of artifacts used (if applicable): *(For example, Python 3.6, Keras 2.1.2, Tensorflow backend

**Comment by mitchellspryn** [comment]
This is expected and desired. Notice how your validation loss is not improving - this means that if you continue to train, your model will overfit. You can see the effects of that by removing the [EarlyStoppingCallback](https://keras.io/callbacks/#earlystopping) from your code and retraining - you will most likely get poor results.

How does the trained model perform? Does it seem to predict the correct angles (in the last cell of the notebook)? How far does the car go before crashing? Note that since we don't have a very large dataset, it's possible that the model converges to a suboptimal minimum. Retraining will help in that case.
